DependencyInjection/SonataAdminExtension.php

```diff
@@ -111,8 +111,7 @@ public function load(array $configs, ContainerBuilder $container)
                         'VIEW' => array('VIEW'),
                         'DELETE' => array('DELETE'),
                         'EXPORT' => array('EXPORT'),
-                        'OPERATOR' => array('OPERATOR'),
-                        'MASTER' => array('MASTER'),
+                        'ALL' => array('ALL'),
```

Comment:
It makes me thing that this: #3839 could have been solved by declaring ALL like:

`'ALL' => array('VIEW', 'EDIT', 'LIST', 'CREATE', 'DELETE', 'EXPORT'),`

Not sure...